DigitalLearningSolutions.Web/Styles/trackingSystem/centreConfiguration.scss

```diff
@@ -1,24 +1,28 @@
-﻿@import "~nhsuk-frontend/packages/core/all";
-
-.centre-detail-image__downsize {
-  display: block;
-  max-width: 500px;
-  max-height: 350px;
-  height: auto;
-  width: auto;
-
-  @include govuk-media-query($until: large-desktop) {
-    max-width: 400px;
-    max-height: 300px;
-  }
-
-  @include govuk-media-query($until: desktop) {
-    max-width: 300px;
-    max-height: 250px;
-  }
-
-  @include govuk-media-query($until: tablet) {
-    max-width: 250px;
-    max-height: 200px;
-  }
-}
+﻿@import "~nhsuk-frontend/packages/core/all";
+
+.centre-detail-image__downsize {
+  display: block;
+  max-width: 500px;
+  max-height: 350px;
+  height: auto;
+  width: auto;
+
+  @include govuk-media-query($until: large-desktop) {
+    max-width: 400px;
+    max-height: 300px;
+  }
+
+  @include govuk-media-query($until: desktop) {
+    max-width: 300px;
+    max-height: 250px;
+  }
+
+  @include govuk-media-query($until: tablet) {
+    max-width: 250px;
+    max-height: 200px;
+  }
+}
+
+.preserve-whitespace{

```

Comment:
This new class is the only actual change. I double checked the file and it is now CRLF.